### PR TITLE
Change to flat ios 7/8 design

### PR DESCRIPTION
**Pull request by eirikfro**
This is a proxy of the new, flat ios 7/8 design.

### CODE REVIEW

File: smart-app-banner.css [hunk 1]
```diff
@@ -39,7 +39,7 @@
 	text-decoration: none;
 	border: 0;
 	border-radius: 14px;
-	-webkit-font-smoothing: subpixel-antialiased;
+    -webkit-font-smoothing: subpixel-antialiased;
```
Comment (dy) [inline smart-app-banner.css:42]: Please stick to original tabs formatting

**Comment by GabLeRoux** [comment]
Looks very good, thanks for this! :+1: @dfcreative this should definitely be merged

**Comment by dy** [comment]
@GabLeRoux as soon as formatting is fixed, with pleasure!

**Comment by GabLeRoux** [comment]
@dfcreative you may want to give a look at #67 :) :tada:

**Pull request merged**